pkg/system/system.go

```diff
@@ -394,6 +395,38 @@ func RunYaml(cmd *cobra.Command, args []string) {
 	util.Panic(p.PrintObj(sys, os.Stdout))
 }
 
+// RunSystemVersionsStatus runs a CLI command
+func RunSystemVersionsStatus(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	klient := util.KubeClient()
+
+	sysKey := client.ObjectKey{Namespace: options.Namespace, Name: options.SystemName}
+	r := NewReconciler(sysKey, klient, scheme.Scheme, nil)
```

Comment:
You don't seem to need the reconciler. Just declare a nbv1.NooBaa instance and KubeCheck it.

Reply:
👍